.github/workflows/ci.yml

```diff
@@ -0,0 +1,225 @@
+name: air.stacks-dotnet-cqrs
+
+# on:
+#   push:
+#     branches:
+#       - master
+#       - main
+#     paths:
+#       - 'build/github/**'
+#       - 'src/**'
+#       - 'deploy/aws/**' # TODO: Review this as new cloud provider targets are used
+#       - 'deploy/k8s/aws/**'  # TODO: Review this as new cloud provider targets are used
+#       - '.github/workflows/ci.yml'
+#   pull_request:
+#     branches:
+#         - master
+#         - main
+
+on: [push]
+
+env:
+  CLOUD_PROVIDER: "aws"
+  TaskctlVersion: '1.4.2'
```

Comment:
Would it be a good idea to have this as an injectable value?

Reply:
This is where the value is injected, it is consumed elsewhere in the pipeline. We would need a separate pipeline file for another cloud provider.